### PR TITLE
Create service and action listener

### PR DESCRIPTION
**Pull request by VuManhHung97**
Create service and action listener notification

https://edu-redmine.sun-asterisk.vn/issues/12521

service

### CODE REVIEW

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            if (intent.getAction() != null) {
+                switch (intent.getAction()) {
+                    case CommonUtils.Action.ACTION_PLAY_AND_PAUSE:
+                        Toast.makeText(this, "ACTION_PLAY_AND_PAUSE", Toast.LENGTH_SHORT).show();
+                        break;
+                    case CommonUtils.Action.ACTION_NEXT:
+                        Toast.makeText(this, "ACTION_NEXT", Toast.LENGTH_SHORT).show();
+                        break;
+                    case CommonUtils.Action.ACTION_PREVIOUS:
+                        Toast.makeText(this, "ACTION_PREVIOUS", Toast.LENGTH_SHORT).show();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java]: hard code

File: app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        if (intent != null) {
+            if (intent.getAction() != null) {
+                switch (intent.getAction()) {
+                    case CommonUtils.Action.ACTION_PLAY_AND_PAUSE:
+                        Toast.makeText(this, "ACTION_PLAY_AND_PAUSE", Toast.LENGTH_SHORT).show();
+                        break;
+                    case CommonUtils.Action.ACTION_NEXT:
+                        Toast.makeText(this, "ACTION_NEXT", Toast.LENGTH_SHORT).show();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java]: hard code

File: app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        mSongBinder = new SongBinder();
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        if (intent != null) {
+            if (intent.getAction() != null) {
+                switch (intent.getAction()) {
+                    case CommonUtils.Action.ACTION_PLAY_AND_PAUSE:
+                        Toast.makeText(this, "ACTION_PLAY_AND_PAUSE", Toast.LENGTH_SHORT).show();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java]: hard code

File: app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        mSongBinder = new SongBinder();
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        if (intent != null) {
+            if (intent.getAction() != null) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/music_65/screen/service/SongService.java]:
```suggestion
            if (intent != null && intent.getAction() != null) {
```

**Comment by VuManhHung97** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**